tidb-cloud/connected-care-announcement.md

```diff
@@ -0,0 +1,78 @@
+|:----------------------------------------------|:-----------------------------|:----------------------------------------------|:-----------------------------------------------|:------------------------------------------|
+| Recommended workloads                          | Personal or starter projects | Workloads under development                   | Workloads under production                     | Business-critical workloads in production |
+| Billing and account inquiries                 | ✔                            | ✔                                             | ✔                                              | ✔                                         |
+| Technical support                             | -                            | ✔                                             | ✔                                              | ✔                                         |
+| Initial response time                         | -                            | Business hours                                | 7x24                                           | 7x24                                      |
+| Connected: Clinic service                     | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: AI chat in IM                      | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM subscription for TiDB Cloud alerts   | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM subscription for ticket updates | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM interaction for support tickets | -                            | -                                             | -                                              | ✔                                         |
+| Technical account manager                     | -                            | -                                             | -                                              | ✔                                         |
+| Pricing                                       | Free                         | Greater of $29 or 10% of total monthly charges | Greater of $699 or 20% of total monthly charges | Contact sales                             |
+
+> **Note**
+>
+> Customers from all four support plans have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.
```

Comment:
@iguoyr Please confirm the [Placeholder] above. Thanks.

Reply:
> @iguoyr Please confirm the [Placeholder] above. Thanks.

Advanced features such as technical account managers, instant messenger integration support, and Clinic services are also available.